src/models/UserModel.ts

```diff
@@ -1,99 +1,27 @@
-import faker from 'faker';
-import { v4 as uuidv4 } from 'uuid';
-
-type GeneralProperties = {
-  id: string;
-  login: string;
-  password: string;
-  age: number;
-}
-
-export type User = GeneralProperties & {
-  isDeleted: boolean;
-}
-
-export class UserManager {
-  private users: User[];
-
-  private generateUsers = (qty: number): User[] => {
-    return Array.from(Array(qty)).map(() => {
-      return {
-        id: uuidv4(),
-        login: faker.name.firstName(),
-        password: faker.lorem.word(),
-        age: faker.datatype.number({
-          min: 18,
-          max: 99
-        }),
-        isDeleted: faker.datatype.boolean()
-      }
-    })
-  }
-
-  constructor (qty: number) {
-    this.users = this.generateUsers(qty);
-  }
-
-  getVisibleUsers = (): User[] => this.users.filter((user: User) => !user.isDeleted);
-
-  findUser = (userId: string): User | undefined =>
-    this.getVisibleUsers().find((user: User) => user.id === userId);
-
-  addUser = (login: string, password: string, age: number): void => {
-    this.users = [{
-      id: uuidv4(),
-      login: String(login),
-      password: String(password),
-      age: Number(age),
-      isDeleted: false
-    },
-    ...this.users]
+import { Sequelize, DataTypes } from 'sequelize';
+import { UserInstance } from '../types';
+
+const connectionString = 'postgres://bqgqthbd:9gOdo8uMOO6yQFufa-ypy2bFDW3SgeXz@dumbo.db.elephantsql.com:5432/bqgqthbd';
```

Comment:
for future suggest to keep those private settings in non-tracked `.env` file. `.env-sample` file should be provided with example configs in this case. Checkout `dotenv ` https://www.npmjs.com/package/dotenv.
This is optional.

Reply:
Used .env file for private settings